bigbluebutton-html5/imports/utils/visibilityEvent.js

```diff
@@ -9,14 +11,25 @@ export default class VisibilityEvent {
     this._handleVisibilityChange = this._handleVisibilityChange.bind(this);
 
     this._registerVisibilityEvent();
+
+    this._onHiddenTimeout = null;
   }
 
   onVisible (f) {
-    this._onVisible = f;
+    this._onVisible = () => {
+      if (!this._isHidden()) {
+        f();
+      }
+    };
   }
 
   onHidden(f) {
-    this._onHidden = f;
+    this._onHidden = () => {
+      if (this._isHidden()) {
```

Comment:
Why do we need to check for `isHidden` on `onHidden`? Is there a case when it's false?
Please change `f` for `callback` or something more meaningful